src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -138,7 +143,14 @@ public RobotMap() {
 			ballioDoorServo);
 		// Climber
 		Component.climber = new Climber(new CANTalon(Port.CANMotor.climbMotorA), new CANTalon(Port.CANMotor.climbMotorB));
-		Component.chassis = new TankDriveShifting("2017-Chassis", Component.leftWheel, Component.rightWheel, Component.shifter);
+		// Shooter
+		SpeedController flywheelMotorA = new CANTalon(Port.CANMotor.flywheelMotorA);
+		SpeedController flywheelMotorB = new CANTalon(Port.CANMotor.flywheelMotorB);
+		CustomEncoder flywheelEncoder = new CANEncoder("FlywheelEncoder", Port.CAN.flywheelEncoder);
```

Comment:
The flywheel encoder will definitely be a CANTalonEncoder. I think this should be fixed before merging, even with a random CANTalon.